### ISSUE TITLE
Upload a Flatpak package to Flathub

### ISSUE THREAD

**Issue by AnderRasoVazquez** [smallhadroncollider/taskell#48]
This will allow everyone to use taskell in any Linux distro. More info below =)

### About Flatpak

>[**Flatpak**](https://flatpak.org/) is a next-generation technology for building and distributing desktop applications on Linux. [...] Flatpak can be used with a total of 22 distros See the [complete list](https://flatpak.org/setup/).

### About Flathub

>[**Flathub**](https://flathub.org/home) is a centralized repository of Flatpak apps. It provides a single point of entry for distributing apps on the most popular Linux distributions. It has a growing number of users, with some Linux distributions enabling it by default, and simple setup steps for those that don’t.

>Once an app is included in Flathub, it can be easily installed by users, either using a graphical software center or the flatpak command line. It is also possible to provide a link that will open the app in a software center.

>Flathub also provides a means for app updates to be distributed to users—once an app is included, its developers can make new releases which will be automatically distributed to users.

>Finally, Flathub can help with exposure for applications, through software centers where available apps are advertised, and through our own websites and social media channels.]

**Comment by smallhadroncollider** [comment]
I'll look into this.

Having a quick glance at their documentation it's not abundantly clear what I need to do. Something about a manifest file, but not quite sure what that's referring to.

Do you have any experience of submitting things to Flathub yourself?

**Comment by AnderRasoVazquez** [comment]
No, but My Ubuntu based distro (elementary OS) has adopted Flatpak as the main package format so I want to learn how to do it too. I'm looking at [Flatpak's twitter](https://twitter.com/FlatpakApps) and I found this [blog post](https://opensource.com/article/19/10/how-build-flatpak-packaging?sc_cid=70160000001273HAAQ) on how to build a Flatpak.

Once the manifest file is done the process to publish it on Flathub looks really easy: [**instructions**](https://github.com/flathub/flathub/wiki/App-Submission#how-to-submit-an-app).

Hope this helps.

**Comment by smallhadroncollider** [comment]
That blog post is very useful. Thanks!
